Simplified/WelcomeScreen/NYPLWelcomeEULAViewController.swift

```diff
@@ -43,6 +43,7 @@ class NYPLWelcomeEULAViewController : UIViewController {
     self.webView.navigationDelegate = self
     self.view.addSubview(self.webView)
 
+    // We're defining rejectButton & acceptButton but not using them anywhere
```

Comment:
that's true... I wonder why Xcode is not letting us know, I suppose because we set the titleLabel...